utils/products.js

```diff
@@ -1,152 +1,676 @@
+    productName: "managed-identity-wallet",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/managed-identity-wallet"],
     committers: [
-      "https://github.com/ds-jkreutzfeld",
-      "https://github.com/ds-jhartmann",
-      "https://github.com/ds-mkanal",
-      "https://github.com/jzbmw"
+      "https://github.com/borisrizov-zf",
+      "https://github.com/nitin-vavdiya",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Item Relationship Service Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-identity-wallet Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "managed-service-orchestrator",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-service-orchestrator",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-service-orchestrator Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-simple-data-exchanger",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-simple-data-exchanger",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-simple-data-exchanger-backend",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger-backend Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-simple-data-exchanger-frontend",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger-frontend Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Policy Hub",
+    productDescription:
+      "The Policy Hub enables data providers, consumers and app providers to access a single-point-of-truth for Catena-X policies, attributes and templates for policy rules.",
+    githubRepo: ["https://github.com/eclipse-tractusx/policy-hub"],
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/Phil91",
+      "https://github.com/tom-rm-meyer-ISST",
```

Comment:
Please remove me from here

Reply:
policy hub was already correctly added, please don't remove it: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/968/files#diff-5c39f3e132d6486132f6c3b616116ec70bb0c8a661a364316f6cbcc299a5c3eeL138

Reply:
I think @tom-rm-meyer-ISST only wants to remove his name from the list instead of the whole product.

Reply:
True. I only would add the current maintainers to the list. I just contributed a small pr :)